src/backend/distributed/sql/udfs/create_distributed_function/latest.sql

```diff
@@ -1,7 +1,11 @@
-CREATE OR REPLACE FUNCTION create_distributed_function(function_name regprocedure)
+CREATE OR REPLACE FUNCTION create_distributed_function(function_name regprocedure,
+						       distribution_arg_name text DEFAULT '',
```

Comment:
why not DEFAULT NULL?

Reply:
Just because it complicates the `C` code a bit, you'd need to use `PG_ARGISNULL` before accessing the value. But, that's  actually a good practice, otherwise if the users passes `NULL` the function crashes.  So, fixing it.